bundles/core/src/main/java/com/adobe/cq/commerce/core/components/models/retriever/AbstractCategoriesRetriever.java

```diff
@@ -166,6 +167,19 @@ protected String generateQuery(List<String> identifiers) {
     protected void populate() {
         GraphqlResponse<Query, Error> response = executeQuery();
         Query rootQuery = response.getData();
+        rootQuery.getCategoryList();
```

Comment:
Is there any particular reason you call `getCategoryList` twice?

Reply:
No. I left that line there by mistake when I added the sorting stuff.
Will remove it right away